src/test/java/com/jalasoft/sdfc/steps/CommonSteps.java

```diff
@@ -87,4 +87,16 @@ public void iValidateTheOpportunityIsNotInTheMenu(String name, String menuName)
         navBar.clickOnTabNameArrow(Item.valueOfItem(menuName));
         assertFalse(navBarMenu.isMenuItemVisible(name));
     }
+
+    @Then("I validate the create form an error message says {string}")
+    public void iValidateTheCreateFormAnErrorMessageSays(String message) {
+        assertEquals(message, basicForm.getErrorMessageAfterClickSave());
+    }
+
+    @And("I validate these fields are required to complete in the form")
```

Comment:
I would change this steps text as `I verify that the following fields are required in the form:`

Reply:
updated